### ISSUE TITLE
Encrypt validator keys

### ISSUE THREAD

**Issue by sug0** [anoma/namada#3251]
Allow encrypting validator keys in `wallet.toml`.

**Comment by sug0** [comment]
cc @cwgoes

**Comment by cwgoes** [comment]
It's not a hard requirement for the release candidate, but we should implement it ASAP.

**Comment by cwgoes** [comment]
@sug0 Which keys are you talking about specifically?

**Comment by sug0** [comment]
@cwgoes these keys, which also live in `wallet.toml`
https://github.com/anoma/namada/blob/f5c3b708018c38bb8768da57ce3199a208ff9e12/crates/sdk/src/wallet/store.rs#L38-L45

**Comment by sug0** [comment]
yeah, they're currently only required by the eth bridge code, which is deactivated anyway

**Comment by brentstone** [comment]
What are thoughts on removing these keys altogether?

**Comment by sug0** [comment]
that answer depends on the premise that the eth bridge is never gonna be enabled. 🤷 we could always add these keys back to the wallet, but idk. @cwgoes

**Comment by cwgoes** [comment]
Let's just leave this issue for now.
